web/src/components/core/InstallationFinished.jsx

```diff
@@ -26,32 +26,61 @@ import {
   EmptyStateBody,
   EmptyStateHeader,
   EmptyStateIcon,
+  ExpandableSection,
+  Hint,
+  HintBody,
 } from "@patternfly/react-core";
 
-import { Page } from "~/components/core";
+import { If, Page } from "~/components/core";
 import { Center, Icon } from "~/components/layout";
 import { useInstallerClient } from "~/context/installer";
+import { EncryptionMethods } from "~/client/storage";
 import { _ } from "~/i18n";
 
+const TpmHint = () => {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <Hint className="tpm-hint">
```

Comment:
As you can see, I solved the `width` (a.k.a. `block-size`) issue with a temporary CSS class.

I have doubts (too long story for writing it here and now) that this combination of a Hint and ExpandableSection is the best approach for this kind of reminders. But I have neither, a clear idea how to make it better now nor the time for cooking some proposals. IMHO, it is good as it is for releasing it by now. Improvements can wait a bit.

Reply:
Actually it was not fixed when I wrote the comment above. See https://github.com/openSUSE/agama/pull/995/commits/30fe3bcda8647d02502da1f3c3bc5cdf01cf74ba